cli/src/main/java/org/jboss/as/cli/impl/aesh/cmd/security/model/InteractiveSecurityBuilder.java

```diff
@@ -168,23 +300,35 @@ public void buildRequest(CommandContext ctx, boolean buildRequest) throws Except
             }
         }
 
-        // then prompt for dn
-        if (dn == null) {
-            dn = new DNWizard().buildDN();
-        }
+        if(useLetsEncrypt) {
+            //then prompt for domainName
+            while (domainNames == null || domainNames.size() < 1) {
+                String domains = commandInvocation.inputLine(new Prompt("Your domain name(s) (must be accessible by the Let's Encrypt server at 80 & 443 ports) [example.com,second.example.com]: "));
+                domainNames = parseItemsFromString(domains, "example.com,second.example.com");
```

Comment:
Showing these examples is ok so user know what format is expected but we shouldn't use these as default values.